src/simplify.rs

```diff
@@ -88,18 +98,21 @@ pub fn simplify(mut input: String, is_chat_message: bool) -> (String, bool, Opti
         if lines.iter().all(|it| it.trim().is_empty()) {
             render_message(original_lines, false)
         } else {
+            is_cut = is_cut || has_nonstandard_footer || bottom_quote.is_some();
             render_message(lines, has_nonstandard_footer || bottom_quote.is_some())
         }
     };
 
     if !is_chat_message {
         top_quote = top_quote.map(|quote| {
             let quote_lines = split_lines(&quote);
-            let quote_lines = remove_message_footer(&quote_lines);
+            let (quote_lines, footer_removed) = remove_message_footer(&quote_lines);
+            is_cut = is_cut || footer_removed;
```

Comment:
Not sure removed *standard* footer should trigger "read more" button, this is probably an overkill.

Reply:
the standard footer may contain an address, phone numbers etc. - if you do not have another email-client, you may want to access these infos somehow.

Reply:
There is an idea by @adbenitez to save the last footer as a "status" and display it in the user profile. Once it is implemented, "read more" can be removed in this case I think.